Xamarin.Forms.Xaml/Properties/AssemblyInfo.cs

```diff
@@ -5,6 +5,7 @@
 [assembly: InternalsVisibleTo("Xamarin.Forms.Xaml.UnitTests")]
 [assembly: InternalsVisibleTo("Xamarin.Forms.Build.Tasks")]
 [assembly: InternalsVisibleTo("Xamarin.Forms.Xaml.Design")]
+[assembly: InternalsVisibleTo("Xamarin.Forms.Loader")]
```

Comment:
you shouldn't be using a namespace starting with `Xamarin.Forms`

you shouldn't also poke into internal APIs, as we might change them, and that would break your tool